### ISSUE TITLE
Accessibility Audit: Placeholder text on filter modal has insufficient contrast ratio

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#933]
Sufficient contrast is defined as having a contrast ratio of 4.5:1 or higher, except for large text (18 pt and above), which can have a contrast ratio of 3:1. Looking at the filter modal on the [Listings](https://detroit-public-dev.netlify.app/listings) page, the Color contrast ratio of placeholder text is around 2.05:1.

![filter_modal_placeholder](https://user-images.githubusercontent.com/394908/147709944-a02399de-1e6d-400f-b0b7-b70581482985.png)
